source/GStreamerWebAudioPlayerClient.cpp

```diff
@@ -140,6 +140,7 @@ bool GStreamerWebAudioPlayerClient::open(GstCaps *caps)
                 {
                     GST_ERROR("GetDeviceInfo failed, could not process samples");
                 }
+                m_frameSize = (pcm.sampleSize * pcm.channels) / 8;
```

Comment:
magic number, constexpr int?

Reply:
Used the system wide macro CHAR_BIT, like I have done in rialto.